### ISSUE TITLE
Prepend stackable variant before DEFAULT utilities CSS

### ISSUE THREAD

**Issue by navin-moorthy** [tailwindlabs/tailwindcss-jit#116]
### What version of @tailwindcss/jit are you using?

v0.1.3

### What version of Node.js are you using?

v15.5.1

### What browser are you using?

Chrome

### What operating system are you using?

macOS

### Reproduction repository

https://github.com/navin-moorthy/tailwindcss-jit/pull/1

In our recent project [Renderlesskit-react-tailwind](https://github.com/timelessco/renderlesskit-react-tailwind), we make use of the Tailwind's variantOrder `color: ["lib", "DEFAULT", "responsive"],` to prepend our custom variant before the default utilities.

Generated output css in [Renderlesskit-react-tailwind](https://github.com/timelessco/renderlesskit-react-tailwind)
```
lib\:text-red-500 {
  --tw-text-opacity: 1;
  color: rgba(239, 68, 68, var(--tw-text-opacity));
}

text-red-500 {
  --tw-text-opacity: 1;
  color: rgba(239, 68, 68, var(--tw-text-opacity));
}
```

But with the `tailwind-jit`, the `variantOrder` is not obeyed. Because of the ability to stack all variants after the `DEFAULT` utilities, our `lib` variant utilities always gets generated in the end.

Generated output css with `tailwind-jit`
```
text-red-500 {
  --tw-text-opacity: 1;
  color: rgba(239, 68, 68, var(--tw-text-opacity));
}

lib\:text-red-500 {
  --tw-text-opacity: 1;
  color: rgba(239, 68, 68, var(--tw-text-opacity));
}

```

But I have managed to find a hacky way to acheive this with tailwind-jit - [PR](https://github.com/navin-moorthy/tailwindcss-jit/pull/1)

- Increased the default utilites Bigint values & decreased the `lib` variant to `0n`

Generated output css with a Bigint hack in `tailwind-jit`
```
lib\:text-red-500 {
  --tw-text-opacity: 1;
  color: rgba(239, 68, 68, var(--tw-text-opacity));
}

text-red-500 {
  --tw-text-opacity: 1;
  color: rgba(239, 68, 68, var(--tw-text-opacity));
}
```

I would love to see this feature implemented if it's possible via some options.

**Comment by adamwathan** [comment]
Yeah this is something we need to figure out still for sure, will work on it 👍🏻
